cleanup-after-each.js

```diff
@@ -0,0 +1 @@
+afterEach(require('./src').cleanup)
```

Comment:
I think this should be requireing `./lib` so that it picks up the compiled version after publishing.